drivers/cleanup.py

```diff
@@ -3206,6 +3206,10 @@ def gc(session, srUuid, inBackground, dryRun=False):
 
 
 def start_gc(session, sr_uuid):
```

Comment:
Why is this code being retained? I don't see any callers, and the implementation of get_state implies that we only expect the systemd version to be called?

Reply:
Mainly to make it easy for someone who wants to revert to a non-systemd method of doing things to do so
(yes, they'd have to reimplement get-state but the original method of doing so has lock-ordering issues, so leaving it would not have been a service)